src/main/java/sparksoniq/spark/iterator/flowr/LetClauseSparkIterator.java

```diff
@@ -36,25 +38,113 @@
 import java.util.List;
 
 public class LetClauseSparkIterator extends FlowrClauseSparkIterator {
```

Comment:
I think the FlworClauseSparkIterator may no longer be needed because the overriden code remaining there will never be reached. We can directly inherit from RuntimeTupleIterator.